src/dapp/src/shared/IPFSGateway.ts

```diff
@@ -1,6 +1,14 @@
+enum VoteType {
```

Comment:
The `VoteType` enum has been defined in 3 separate places. Is that deliberate?

Reply:
Deliberate, but I've added a comment to link them.